test/test_integration_pumactl.rb

```diff
@@ -89,7 +89,7 @@ def test_phased_restart_cluster
 
     _, status = Process.wait2(@pid)
     assert_equal 0, status
-    assert_operator Time.now - start, :<, (DARWIN ? 8 : 6)
+    assert_operator Time.now - start, :<, (DARWIN ? 8 : 7)
```

Comment:
Maybe this test should be using `Process.clock_gettime(Process::CLOCK_MONOTONIC)`? https://blog.dnsimple.com/2018/03/elapsed-time-with-ruby-the-right-way/

(but giving the test more time seems fine to me)

Reply:
Done